backend/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.9 AS development
+WORKDIR /usr/src/aof
+ENV PYTHONPATH=/usr/src/aof:$PYTHONPATH
```

Comment:
why do we need this?

Reply:
Are you referring to the Dockerfile or a specific line or lines?

Reply:
to line n 3

Reply:
https://docs.python.org/3.9/using/cmdline.html#envvar-PYTHONPATH
I tend to use absolute imports. Adding the top-level folder to PYTHONPATH will make python packages that are inside discoverable for python.

Reply:
wouldn't it be enough to use relative imports?

Reply:
>Absolute imports are preferred because they are quite clear and straightforward. It is easy to tell exactly where the imported resource is, just by looking at the statement. Additionally, absolute imports remain valid even if the current location of the import statement changes. In fact, PEP 8 explicitly recommends absolute imports.

https://realpython.com/absolute-vs-relative-python-imports/

Reply:
>Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
but even if we do absolute imports, we shouldn't need to extend the pythonpath as long as we launch the python script from the root of the project. Were you thinking to launch it from somewhere else?

Reply:
I figured we'd have something like this in the dev env
```console
$ tree /usr/src/aof
/usr/src/aof/
├── docker-compose.dev.yaml
├── Dockerfile
├── poetry.lock
├── pyproject.toml
└── src
    └── main.py

1 directory, 5 files
```
I am happy to remove that line and see if we run into an issue down the line.
I'll amend the commit.

Reply:
Removed. Please resolve the conversation if we are good.